### ISSUE TITLE
Dataset cards: missing template for the YAML tags

### ISSUE THREAD

**Issue by lhoestq** [huggingface/hub-docs#222]
On this page I think it would be nice to have a template of the YAML tags users can add to their dataset cards:
https://huggingface.co/docs/hub/datasets-cards

Overall the page feels empty compared to the one for models ^^'

The full list of supported tags is currently this one:
```
    annotations_creators: List[str]
    language_creators: List[str]
    language: List[str]
    license: Union[str, List[str]]
    license_details: str
    multilinguality: List[str]
    pretty_name: str
    size_categories: List[str]
    source_datasets: List[str]
    task_categories: List[str]
    task_ids: List[str]
    paperswithcode_id: str
    train-eval-index: List[Dict]
    configs: List[str]
    extra_gated_fields: Dict
    extra_gated_prompt: str
```

Maybe @stevhliu or @NimaBoscarino can help with this ?

EDIT: added str to license

**Comment by julien-c** [comment]
❤️ ❤️ 

note that on Hub side `license:` can be `str` or `List[str]` so i'd recommend documenting the same

**Comment by NimaBoscarino** [comment]
Sounds great to me! I can help with this later this week :)

**Comment by stevhliu** [comment]
Closed by #254 :)
